src/lang/en.json

```diff
@@ -86,6 +86,39 @@
     "ignoreTLSError": "Ignore TLS/SSL errors for HTTPS websites",
     "ignoreTLSErrorGeneral": "Ignore TLS/SSL error for connection",
     "upsideDownModeDescription": "Flip the status upside down. If the service is reachable, it is DOWN.",
+    "wsIgnoreHeadersDescription": "The websocket upgrade succeeds, but the server does not reply with Sec-WebSocket-Accept header.",
+    "Ignore Sec-WebSocket-Accept header": "Ignore Sec-WebSocket-Accept header",
+    "wamp": "WAMP (The WebSocket Application Messaging Protocol)",
+    "sip": "WebSocket Transport for SIP (Session Initiation Protocol)",
+    "notificationchannel-netapi-rest.openmobilealliance.org": "OMA RESTful Network API for Notification Channel",
+    "wpcp": "Web Process Control Protocol (WPCP)",
+    "amqp": "Advanced Message Queuing Protocol (AMQP) 1.0+",
+    "jsflow": "jsFlow pubsub/queue protocol",
+    "rwpcp": "Reverse Web Process Control Protocol (RWPCP)",
+    "xmpp": "WebSocket Transport for the Extensible Messaging and Presence Protocol (XMPP)",
+    "ship": "SHIP - Smart Home IP",
+    "mielecloudconnect": "Miele Cloud Connect Protocol",
+    "v10.pcp.sap.com": "Push Channel Protocol",
+    "msrp": "WebSocket Transport for MSRP (Message Session Relay Protocol)",
+    "bfcp": "WebSocket Transport for BFCP (Binary Floor Control Protocol)",
+    "sldp.softvelum.com": "Softvelum Low Delay Protocol",
+    "opcua+uacp": "OPC UA Connection Protocol",
+    "opcua+uajson": "OPC UA JSON Encoding",
+    "v1.swindon-lattice+json": "Swindon Web Server Protocol (JSON encoding)",
+    "v1.usp": "USP (Broadband Forum User Services Platform)",
+    "coap": "Constrained Application Protocol (CoAP)",
+    "webrtc.softvelum.com": "Softvelum WebSocket signaling protocol",
+    "cobra.v2.json": "Cobra Real Time Messaging Protocol",
+    "drp": "Declarative Resource Protocol",
+    "hub.bsc.bacnet.org": "BACnet Secure Connect Hub Connection",
+    "dc.bsc.bacnet.org": "BACnet Secure Connect Direct Connection",
+    "jmap": "WebSocket Transport for JMAP (JSON Meta Application Protocol)",
+    "t140": "ITU-T T.140 Real-Time Text",
+    "done": "Done.best IoT Protocol",
```

Comment:
Instead of adding translation keys like "done" for the subprotocols, please just use "Done.best IoT Protocol". This is much less confusing